### PR TITLE
feat(nova): impl name, email, and phone number inputs

### PR DESCRIPTION
**Pull request by moalidv**

### CODE REVIEW

File: apps/nova/src/components/Settings/UploadPhoto.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const UploadPhoto: React.FC<{
   const ref = useRef<HTMLInputElement>(null);
 
   return (
-    <div className="flex flex-row gap-6">
+    <div className="flex flex-row gap-6 mb-8">
```
Comment (neuodev) [inline apps/nova/src/components/Settings/UploadPhoto.tsx:16]: Don't set the `mb` here! The parent should control the spacing.

File: apps/nova/src/components/Settings/Personal.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    const form = useForm<IForm>()
+
+
+    // const nameRules = useValidateName()
+    const nameRules = validateText({
+        regex: USER_NAME_REGEX, value: form.getValues("name"), length: { min: 2, max: 50 }, errors: {
+            match: intl("error.name.invalid"), min: intl("error.name.length.short"), max: intl("error.name.length.long"),
+        }
+    })
+    const emailRules = useValidateEmail(true);
```
Comment (neuodev) [inline apps/nova/src/components/Settings/Personal.tsx:29]: Please use the functions defined in `@litespace/sol` and don't use old logic.

**Comment by neuodev** [comment]
🚀 [Nova](https://nova-gmy247n3g-litespace-cores-projects.vercel.app) 
🚀 [Luna](https://luna-6beiohmh3-litespace-cores-projects.vercel.app) 
🚀 [Nebula](https://nebula-odn0d5lj7-litespace-cores-projects.vercel.app) 
🚀 [Dashboard](https://dashboard-ns9106qi2-litespace-cores-projects.vercel.app)

File: apps/nova/src/components/Settings/Field.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { Field as CustomField } from "@litespace/luna/Form";
+
```
Comment (moalidv) [inline apps/nova/src/components/Settings/Field.tsx:2]: empty line

**Comment by neuodev** [comment]
🚀 [Nova](https://nova-khvbfsd05-litespace-cores-projects.vercel.app) 
🚀 [Luna](https://luna-l61j6a103-litespace-cores-projects.vercel.app) 
🚀 [Nebula](https://nebula-8ibzryd1j-litespace-cores-projects.vercel.app) 
🚀 [Dashboard](https://dashboard-t010ds07z-litespace-cores-projects.vercel.app)

**Comment by neuodev** [comment]
🚀 [Nova](https://nova-khvbfsd05-litespace-cores-projects.vercel.app) 
🚀 [Luna](https://luna-l61j6a103-litespace-cores-projects.vercel.app) 
🚀 [Nebula](https://nebula-3wbtcze2i-litespace-cores-projects.vercel.app) 
🚀 [Dashboard](https://dashboard-f2k9yrcgc-litespace-cores-projects.vercel.app)

**Comment by neuodev** [comment]
🚀 [Nova](https://nova-khvbfsd05-litespace-cores-projects.vercel.app) 
🚀 [Luna](https://luna-l61j6a103-litespace-cores-projects.vercel.app) 
🚀 [Nebula](https://nebula-3wbtcze2i-litespace-cores-projects.vercel.app) 
🚀 [Dashboard](https://dashboard-jmbqjjlzc-litespace-cores-projects.vercel.app)

**Review by neuodev — APPROVED** [review]

**Pull request merged**